dev-java/openjfx/openjfx-21_p31.ebuild

```diff
@@ -0,0 +1,237 @@
+# Copyright 2019-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV="${PV/_p/+}"
+MY_PV2="${MY_PV%+*}"
+SLOT="${MY_PV%%[.+]*}"
+
+# https://docs.gradle.org/current/userguide/compatibility.html
+EGRADLE_VER="8.5"
+
+inherit flag-o-matic java-pkg-2 multiprocessing toolchain-funcs
+
+DESCRIPTION="Java OpenJFX client application platform"
+HOMEPAGE="https://openjfx.io"
+
+SRC_URI="
+	https://github.com/openjdk/jfx/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://downloads.gradle.org/distributions/gradle-${EGRADLE_VER}-bin.zip
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-sandbox/7.7.3/lucene-sandbox-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-grouping/7.7.3/lucene-grouping-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-queryparser/7.7.3/lucene-queryparser-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-queries/7.7.3/lucene-queries-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-core/7.7.3/lucene-core-7.7.3.jar
+	https://repo1.maven.org/maven2/org/antlr/antlr4/4.7.2/antlr4-4.7.2-complete.jar
```

Comment:
Doesn't it work with [dev-java/antlr:4](https://packages.gentoo.org/packages/dev-java/antlr)?

Reply:
no, the current installation is buggy because of how deps among antlr* packages are written.